app/views/layouts/application.html.erb

```diff
@@ -203,10 +203,6 @@
             <%= t("layouts.application.footer.gems_served_by") %>
             <span class="t-hidden">Fastly</span>
           </a>
-          <a class="footer__sponsor" href="https://travis-ci.org/" target="_blank">
-            <%= t("layouts.application.footer.tested_by") %>
-            <span class="t-hidden">Travis CI</span>
-          </a>
```

Comment:
I think we can do this in a separate PR. We need to update the sprite image in the background or else the order of links will get messed up.

Reply:
:+1: